File: DKS-SPA/src/app/core/_models/dev-gate-log-data-log.ts
import { ModelInterface } from "./interface/model-interface";

export class DevGateLogDataLog implements ModelInterface {
  seq: number;
  reason: string;
  updater: string;
  flag: string;


  constructor(){
    this.seq = 0;
    this.reason = "";
    this.updater = "";
    this.flag = "";
  }
}

File: DKS-API/Models/DKS/DevGateLogDataLog.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class DevGateLogDataLog
    {
        [Key]
        public int SEQ { get; set; }

        public string REASON { get; set; }

        public string UPDATER { get; set; }

        public string? FLAG { get; set; }

        public string UPDATETIME { get; set; }
        
    }
}